Raven.Tests.Helpers/RavenTestBase.cs

```diff
@@ -865,5 +962,65 @@ protected static void DeployNorthwind(DocumentStore store, string databaseName =
 			var request = requestFactory.CreateHttpJsonRequest(new CreateHttpJsonRequestParams(null, url + "/studio-tasks/createSampleData", "POST", store.DatabaseCommands.PrimaryCredentials, store.Conventions));
 			request.ExecuteRequest();
 		}
+
+        private static void StoreSeedData(IEnumerable<IEnumerable> seedData,
```

Comment:
I thought so too - when dynamic magic is involved it's extremely hard for developers to discover what it can do.

I'll go and add some new overloads to make it more awesome for developers. I'll update the PR now :)